doc/INSTALL.md

```diff
@@ -52,8 +52,8 @@ production workload and a `dev` installation for experimentation, the below comm
 two dataplanes, one into the `pod` and another one into the `dev` namespace.
 
 ```console
-helm install stunner stunner/stunner-prod --create-namespace --namespace=stunner-prod
-helm install stunner stunner/stunner-dev --create-namespace --namespace=stunner-dev
+helm install stunner stunner/stunner --create-namespace --namespace=stunner-prod
```

Comment:
What about using `helm install stunner-prod stunner/stunner --create-namespace --namespace=stunner-prod` here? I guess that would somehow differentiate the two installs right in the release name, no?

Reply:
good idea

Reply:
Since we don't have a stunner-prod chart built, it would not work anyway. Release name and namespace should make the difference.